tests/unit/CertificateGenerationService.unitTest.ts

```diff
@@ -1244,18 +1245,72 @@ describe("Certificate Generation Service", () => {
             jest.resetAllMocks();
           });
         });
+        context("with a test station that is an empty string", () => {
```

Comment:
can you have reword this please, something like when the test station is an empty string, same for other its please

Reply:
Have updated the wording so it's a bit more readable. 

Left the 'with a ....' as that is consistent with every test in this context and follows the 'test isTestStationWelsh method with a ...' naming conventions that are already there.